main.py

```diff
@@ -96,7 +136,7 @@ def main():
                     logger.info("No Giving partners left to process")
                     return
                 logger.info(
-                    f"No row exists with provided id: {args.id} in the donee_info / giving patners table"  # log this # pylint: disable=line-too-long
+                    f"No active and unregistered GP exists with the provided id: {args.id} in the donee_info / giving patners table"  # pylint: disable=line-too-long
```

Comment:
We are looking for active and registered
```suggestion
                    f"No active and registered GP exists with the provided id: {args.id} "  # pylint: disable=line-too-long
```